package.json

```diff
@@ -244,7 +244,7 @@
         ]
       },
       {
-        "permissionName": "ui-eholdings.titles-packages.create-delete",
+        "permissionName": "ui-eholdings.titles-packages.manage",
```

Comment:
Maybe call it `title-packages.create-delete.execute` or `title-packages.create-delete.manage`?
Again, without `create-delete` it's impossible to tell what we are managing or executing.